spec/hydra/pcdm/models/versioned_file_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+
+describe Hydra::PCDM::File do
+
+  let(:file) { Hydra::PCDM::VersionedFile.new }
+  let(:reloaded) { Hydra::PCDM::VersionedFile.new(file.uri) }
+
+  it "has_many_versions" do
+    expect(file.class).to be_versionable
+  end
+
+  describe "when saving" do
+    it "sets the PCDMTerms.File RDF type" do
+      file.content = 'stuff'
+      expect(file.save).to be true
+      expect(reloaded.metadata_node.query(predicate: RDF.type, object: RDFVocabularies::PCDMTerms.File).map(&:object)).to eq [RDFVocabularies::PCDMTerms.File]
+    end
+  end
+
+  describe "#label" do
+    it "saves a label" do
+      file.content = 'stuff'
+      file.label = 'foo'
+      expect(file.label).to eq ['foo']
+      expect(file.save).to be true
+      expect(reloaded.label).to eq ['foo']
+    end
+  end
+
+  describe "technical metadata" do
+    let(:date_created) { Date.parse "Fri, 08 May 2015 08:00:00 -0400 (EDT)" }
+    let(:date_modified) { Date.parse "Sat, 09 May 2015 09:00:00 -0400 (EDT)" }
+    let(:content) { "hello world" }
+    let(:file) { Hydra::PCDM::File.new.tap { |ds| ds.content = content } }
+    it "saves technical metadata" do
+      file.file_name = "picture.jpg"
```

Comment:
Consider moving lines 36-41 into a `before` block to keep setup and test execution separate.